docs/user-guides/cdc-ingestion.md

```diff
@@ -8,4 +8,500 @@ menu:
         parent: User Guides
         weight: 400
 ---
-# CDC Ingestion
\ No newline at end of file
+# CDC Ingestion
+CDC stands for Change Data Capture, which is a broad concept, as long as it can capture the change data, it can be called CDC. Flink CDC is a Log message-based data capture tool, all the inventory and incremental data can be captured. Taking MySQL as an example, it can easily capture Binlog data through Debezium and process the calculations in real time to send them to the Arctic data lake. The Arctic data lake can then be queried by other engines.
+
+This section will show how to practice one table into the lake and multiple tables into the lake for both Native-Iceberg and Mixed-Iceberg format.
+## One table into the lake
+### Native Iceberg format
+The following example will show how MySQL CDC data is written to a Native-Iceberg table.
+```sql
+CREATE TABLE products (
+    id INT,
+    name STRING,
+    description STRING,
+    PRIMARY KEY (id) NOT ENFORCED
+) WITH (
+    'connector' = 'mysql-cdc',
+    'hostname' = 'localhost',
+    'port' = '3306',
+    'username' = 'root',
+    'password' = '123456',
+    'database-name' = 'mydb',
+    'table-name' = 'products'
+);
+  
+CREATE CATALOG iceberg_hadoop_catalog WITH (
+    'type'='iceberg',
+    'catalog-type'='hadoop',
+    'warehouse'='hdfs://nn:8020/warehouse/path',
+    'property-version'='1'
+);
+
+CREATE TABLE IF NOT EXISTS `iceberg_hadoop_catalog`.`default`.`sample` (
+    id INT,
+    name STRING,
+    description STRING,
+    PRIMARY KEY (id) NOT ENFORCED
+);
+
+insert into `iceberg_hadoop_catalog`.`default`.`sample` select * from products;
+```
+
+### Mixed Iceberg format
+The following example will show how MySQL CDC data is written to a Mixed-Iceberg table.
+```sql
+CREATE TABLE products (
+    id INT,
+    name STRING,
+    description STRING,
+    PRIMARY KEY (id) NOT ENFORCED
+) WITH (
+    'connector' = 'mysql-cdc',
+    'hostname' = 'localhost',
+    'port' = '3306',
+    'username' = 'root',
+    'password' = '123456',
+    'database-name' = 'mydb',
+    'table-name' = 'products'
+);
+
+CREATE CATALOG arctic_catalog WITH (
+    'type'='arctic',
+    'metastore.url'='thrift://<ip>:<port>/<catalog_name_in_metastore>'
+); 
+
+CREATE TABLE IF NOT EXISTS `arctic_catalog`.`db`.`test_tb`(
+    id INT,
+    name STRING,
+    description STRING,
+    PRIMARY KEY (id) NOT ENFORCED
+);
+
+insert into `arctic_catalog`.`db`.`test_tb` select * from products;
+```
+
+## Multiple tables into the lake
+### Native Iceberg format
+The following example will show how to write CDC data from multiple MySQL tables into the corresponding Native-Iceberg table.
+```java
+import com.ververica.cdc.connectors.mysql.source.MySqlSource;
```

Comment:
I think we can remove the import, and add maven or gradle dependency detail at the head of demos